src/web/components/Core/ErrorView.tsx

```diff
@@ -1,23 +1,36 @@
-import { analyticsIdentifier } from '../../utils/errorHelpers';
+import axios from 'axios';
+import { useAsyncError } from 'react-router-dom';
+
+import { ApiError, getErrorHash } from '../../utils/apiError';
+import { analyticsIdentifier, extractMessageFromAxiosError } from '../../utils/errorHelpers';
 
 type ErrorViewProps = {
   message?: string;
   errorId?: string;
   errorHash?: string;
 };
 
-function ErrorView({ message, errorId, errorHash }: ErrorViewProps) {
+export function ErrorView({ message, errorId, errorHash }: ErrorViewProps) {
   return (
     <div className='error-content'>
       <img alt='Error icon' src='/uid2-logo.png' />
-      <div>Error</div>
-      <div>
-        {message ??
-          'There was an unexpected error. Please try again. If the problem persists, contact Support and provide the following information: '}
-      </div>
+      <div>There was an unexpected error. Please try again.</div>
```

Comment:
Some minor changes here - I think they make a little more sense. It's probably sensible to always provide the stock text even if we provide a custom message.